client/src/components/backend/Form/HasMany.js

```diff
@@ -155,7 +178,7 @@ export default class FormMakers extends PureComponent {
                 return `${option.attributes.firstName} ${option.attributes.lastName}`;
               }
             }
-            onNew={this.onNew}
+            onNew={this.props.onNew}
```

Comment:
This change introduces a regression that breaks the ability to add a new maker to a project on the project collaborators view in the backend. Prior to this change, when the predictiveInput triggers the "onNew" callback, it's triggering the `onNew` method in the HasMany component. That's important, because that method will call this.props.onNew (the onNew method that was passed to HasMany), and then it will call this.props.onChange, because the assumption is that the new record is also now the selected record.

This change just passes the hasMany onNew prop directly to the predictive input, which breaks this behavior.